plugin/slime.vim

```diff
@@ -77,7 +77,9 @@ function! s:TmuxConfig() abort
   if !exists("b:slime_config")
     let b:slime_config = {"socket_name": "default", "target_pane": ":"}
   end
-  let b:slime_config["socket_name"] = input("tmux socket name or absolute path: ", b:slime_config["socket_name"])
+  if !(exists("g:slime_dont_ask_tmux_socket_name") && g:slime_dont_ask_tmux_socket_name)
```

Comment:
I'm thinking about changing that code to be able to provide a function pointer to do the configuration.

It's a part of the code where everybody seems to have opinions about what's a reasonable default.